renovate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "extends":[
+    "config:base"
+  ],
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/bower_components/**",
+    "**/vendor/**",
+    "**/examples/**"
+  ],
+  "enabled": "true",
+  "baseBranches": ["main"],
+  "packageRules": [
```

Comment:
In your fork there are currently two PRs open for `cluster-api` and `cluster-api/test`. As those are always released together, we can create a group for them, so that one PR can update both:

```
"packageRules": [
    {
      "matchPackageNames": ["sigs.k8s.io/cluster-api", "sigs.k8s.io/cluster-api/test"],
      "groupName": "Cluster API"
    }
  ]
```